george/server.js

```diff
@@ -0,0 +1,116 @@
+var express = require('express');
+var app = express();
+var bodyParser = require('body-parser');
+var mongoose = require('mongoose');
+
+mongoose.connect("mongodb://localhost/gorilla_collection");
+var UserSchema = new mongoose.Schema({
+	name: String,
+	age: Number
+});
+mongoose.model('User', UserSchema);
+var User = mongoose.model('User');
+
+app.use(bodyParser.urlencoded({ extended: true }));
+
+var path = require('path');
```

Comment:
Because of potential weird hoisting behavior, it's always recommended to define variables at the top of a document.